src/extensions/multiple-sort/bootstrap-table-multiple-sort.js

```diff
@@ -599,18 +599,36 @@ BootstrapTable.prototype.onMultipleSort = function () {
     const arr2 = []
 
     for (let i = 0; i < that.options.sortPriority.length; i++) {
-      const order = that.options.sortPriority[i].sortOrder === 'desc' ? -1 : 1
-      let aa = a[that.options.sortPriority[i].sortName]
-      let bb = b[that.options.sortPriority[i].sortName]
+      // get fieldName from multi sort
```

Comment:
remove comments and remove the empty lines between the variables

Reply:
Removed comments and empty lines in latest commit